src/components/Attachments/AttachmentView/propTypes.js

```diff
@@ -22,6 +22,9 @@ const attachmentViewPropTypes = {
 
     /** Handles scale changed event */
     onScaleChanged: PropTypes.func,
+
+    /** Whether this AttachmentView is shown as part of a AttachmentModal */
```

Comment:
```suggestion
    /** Whether this AttachmentView is shown as part of an AttachmentModal */
```
NAB

Reply:
Yes sorry forgot to update this one earlier.